domain/cloud/state/state.go

```diff
@@ -465,9 +465,6 @@ FROM   cloud
 		"cloud_name":           name,
 	}
 	iter := tx.Query(ctx, loadCloudStmt, args).Iter()
-	if err != nil && !errors.Is(err, sqlair.ErrNoRows) {
-		return nil, errors.Trace(err)
-	}
```

Comment:
This should not be deleted.

Reply:
The above Query does not have an error returned at all.